Duplicate images and simple errors

TomTheTonk opened the pull request:
Conda.rst extra period
dtm.rst removed ``
index.rst removed picture
meshing updated code block
histogram.rst updated code block
rasterize.rst added file name
deleted dtm-qgis-colorize-dtm and georeferencing-run-command

gannon-guess commented on doc/workshop/exercises/analysis/rasterize/rasterize.rst:
I believe that is the file location, not the file name

gannon-guess commented on doc/workshop/exercises/python/histogram.rst:
add a new line after histogram.json

gannon-guess commented on doc/workshop/exercises/analysis/meshing/meshing.rst:
This change was also already made

gannon-guess commented on doc/workshop/exercises/analysis/dtm/dtm.rst:
return the images/dtm-qgis-colorize-dtm.png
Line 104-107

gannon-guess commented on doc/workshop/conda.rst:
This change has already been made

gannon-guess commented on doc/workshop/exercises/georeferencing/index.rst:
This can be removed as well bc I had to make more edits to the file beyond this, so I just included this with that. Should be smoother than trying to rebase this in later.

TomTheTonk committed:
Duplicate images and simple errors

dtm.rst removed ``
index.rst removed picture
meshing updated code block
histogram.rst updated code block
rasterize.rst added file name
deleted georeferencing-run-command
